### PR TITLE
Rename active active to replicated event sourcing

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Review by chbatey — APPROVED** [review]
LGTM

**Comment by chbatey** [comment]
I'll fix the compile issue

File: akka-persistence-testkit/src/test/scala/akka/persistence/testkit/scaladsl/ScalaDslUtils.scala [hunk 1]
```diff
@@ -8,21 +8,17 @@ import akka.actor.typed.ActorRef
 import akka.persistence.testkit.{ Cmd, CommonUtils, EmptyState, Evt, Passivate, Recovered, Stopped, TestCommand }
 import akka.persistence.typed.PersistenceId
 import akka.persistence.typed.RecoveryCompleted
-import akka.persistence.typed.scaladsl.{ Effect, ReplicatedEventSourcedBehavior }
```
Comment (johanandren) [inline akka-persistence-testkit/src/test/scala/akka/persistence/testkit/scaladsl/ScalaDslUtils.scala]: Ah, god damn it, IntelliJ did too much and I reverted but missed that it changed here as well. Thx

**Comment by chbatey** [comment]
It shall never pass

**Pull request merged**